recipes/id3v2lib/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
```

Comment:
The test_v1_package is no longer required for new recipes. You can remove it if you want: https://github.com/conan-io/conan-center-index/discussions/17406